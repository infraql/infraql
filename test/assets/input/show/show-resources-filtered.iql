SHOW RESOURCES 
in google.compute 
like '%instances%'
;