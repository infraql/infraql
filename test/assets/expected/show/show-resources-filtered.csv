instances,compute__v1.instances,Instance
